src/Controllers/ElementalAreaController.php

```diff
@@ -159,6 +161,19 @@ public function apiSaveForm(HTTPRequest $request)
         return HTTPResponse::create($body)->addHeader('Content-Type', 'application/json');
     }
 
+    public function formAction(HTTPRequest $request)
```

Comment:
Can we have a doc block explaining why this method exists please?

Reply:
Thanks for adding the doc block!